.github/workflows/testBuildAndRelease.yml

```diff
@@ -21,9 +21,10 @@ jobs:
   build-and-test:
     uses: ./.github/workflows/buildAndTest.yml
     needs: [get-version]
-    with: 
+    with:
       branch: 'main'
       label: ${{ needs.get-version.outputs.RELEASE_VERSION }}
+      runId: ${{ github.run_id }}
```

Comment:
oh there is it.  
Is this a sha or what?  should we prefix it with the branch or something to make it more easily identifiable?

Reply:
We need only the id of the run that created the vsixes to then use it with `gh run download` command, so we can't add anything else

Reply:
I think we can name it whatever we use if we use the gh cli flag `gh run download -n <name1>`. I'm not that hung up on it.  If it's a problem we can address later